.github/workflows/windows.yml

```diff
@@ -0,0 +1,168 @@
+name: Windows Build
+
+on:
+  pull_request:
+    branches:
+      - main
+
+jobs:
+  windows-spm:
+    runs-on: windows-latest
+
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+          - branch: swift-6.0-release
+            tag: 6.0-RELEASE
+
+    name: SPM ${{ matrix.tag }}
+
+    steps:
+      - uses: compnerd/gha-setup-swift@main
+        with:
+          tag: ${{ matrix.tag }}
+          branch: ${{ matrix.branch }}
+
+      - uses: actions/checkout@v4
+
+      - name: Build
+        run: swift build -v
+
+      - name: Test
+        run: swift test -v
+
+  windows-cmake:
+    runs-on: windows-latest
+
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+          - branch: swift-6.0-release
+            tag: 6.0-RELEASE
+
+    name: CMake ${{ matrix.tag }}
+
+    steps:
+      - uses: compnerd/gha-setup-vsdevenv@main
+
+      - uses: compnerd/gha-setup-swift@main
+        with:
+          tag: ${{ matrix.tag }}
+          branch: ${{ matrix.branch }}
+
+      - uses: actions/checkout@v4
+        with:
+          path: ${{ github.workspace }}/SourceCache/swift-docc
+
+      - uses: actions/checkout@v4
+        with:
+          repository: apple/swift-argument-parser
+          ref: 0fbc8848e389af3bb55c182bc19ca9d5dc2f255b # 1.4.0
```

Comment:
How will we ensure that these commit hashes are up-to-date with the Package.resolved file?

Reply:
We can track the branch if you want, I don't have a strong preference, I just pinned it to the revision since I accidentally opened Package.resolved instead of Package.swift.

Reply:
Ideally it would track the commit hashes of the Package.resolved file so that Windows uses the same versions of all dependencies as all other platforms. That may be unfeasible to implement. If so, I believe that tracking branches instead of commit hashes will require less ongoing maintenance.